generals/config.py

```diff
@@ -12,8 +11,11 @@ class GameConfig(BaseModel):
     general_positions: List[Tuple[int, int]] = None # Positions of generals
     map: str = None # Map layout as string
     replay_file: str = None # File from which to replay the game
-    agent_names: List[str] = None # Names of the agents that will be called to play the game
-    gymnasium_npc: str = "random" # Name of the opponent to be used in gymnasium
+    agents: List = Field(default_factory=list) # List of agents
+
+    class Config:
+        arbitrary_types_allowed = True
+
```

Comment:
This is wild, add TODO to make purpose of this clearer or something.

Reply:
That shit wild